exist-core/src/main/java/org/exist/storage/serializers/Serializer.java

```diff
@@ -191,6 +191,7 @@ public abstract class Serializer implements XMLReader {
     protected Receiver receiver = null;
     protected SAXSerializer xmlout = null;
     protected LexicalHandler lexicalHandler = null;
+	protected Map<Integer, String> useCharacterMaps = null;
```

Comment:
Indentation looks off.

Reply:
Ah. Looks like the file is 50% spaces and 50% tabs, and every editor interprets tab as 4. But git-diff shows tabs as 8. I've changed the new line(s) to use spaces..